### PR TITLE
Add isBetOrRaiseExistOnFollowingActions

### PR DESCRIPTION
**Pull request by ryotogashi**

### CODE REVIEW

**Comment by ryotogashi** [comment]
@axross Can you check when you have time?

File: hooks/usePlayerActions/isBetOrRaiseExistOnFollowingActions.test.ts [hunk 1]
```diff
@@ -0,0 +1,55 @@
+import { GameStreetActionType } from "../../models/GameSituation";
+import isBetOrRaiseExistOnFollowingActions from "./isBetOrRaiseExistOnFollowingActions";
+
+describe("isBetOrRaiseExistOnFollowingActions(gameStreetActions: GameStreetAction[], currentIndex: number)", () => {
+  it("returns true when bet is on following actions", () => {
+    expect(
+      isBetOrRaiseExistOnFollowingActions(
+        [
```
Comment (ryotogashi) [inline hooks/usePlayerActions/isBetOrRaiseExistOnFollowingActions.test.ts:8]: This PlayerAuctions seems wrong for you. but it happens when PlayerAuctions[3].type change Check to Bet.

File: hooks/usePlayerActions/isBetOrRaiseExistOnFollowingActions.ts [hunk 2]
```diff
@@ -0,0 +1,19 @@
+import {
+  GameStreetAction,
+  GameStreetActionType,
+} from "../../models/GameSituation";
+
+export default function isBetOrRaiseExistOnFollowingActions(
+  gameStreetActions: GameStreetAction[],
+  currentIndex: number
+) {
```
Comment (axross) [inline hooks/usePlayerActions/isBetOrRaiseExistOnFollowingActions.ts:9]: Maybe better 🤔

```suggestion
export default function hasBestOrRaiseAfter(
  actions: GameStreetAction[],
  index: number
) {
```

File: hooks/usePlayerActions/isBetOrRaiseExistOnFollowingActions.ts [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+  for (const { type } of gameStreetActions.slice(currentIndex + 1)) {
+    if (
+      type === GameStreetActionType.bet ||
+      type === GameStreetActionType.raise
+    )
+      return true;
+  }
+
+  return false;
```
Comment (axross) [inline hooks/usePlayerActions/isBetOrRaiseExistOnFollowingActions.ts:18]:
```suggestion
  return gameStreetActions
    .slice(currentIndex + 1)
    .some(({ type }) => type === GameStreetActionType.bet || type === GameStreetActionType.raise )
```

**Comment by ryotogashi** [comment]
Thank you! some!!! it is for my first time!

File: hooks/usePlayerActions/hasBestOrRaiseAfter.ts [hunk 4]
```diff
@@ -0,0 +1,16 @@
+import {
+  GameStreetAction,
+  GameStreetActionType,
+} from "../../models/GameSituation";
+
+export default function hasBestOrRaiseAfter(
```
Comment (axross) [inline hooks/usePlayerActions/hasBestOrRaiseAfter.ts:6]:
```suggestion
export default function hasBetOrRaiseAfter(
```

Comment (ryotogashi) [reply hooks/usePlayerActions/hasBestOrRaiseAfter.ts:6]: OMG🙀

**Pull request merged**